db/seeds.rb

```diff
@@ -111,6 +138,10 @@
                   tenant_id: bridge_builders2.id
                   )
 
+johns_waterworks_cotedivore.photos << Photo.create!(
+        image: File.new( "#{Rails.root}/app/assets/images/johns_waterworks.jpg")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Space inside parentheses detected.